backend/app/main.py

```diff
@@ -171,6 +171,18 @@ async def shatter_parent(
     return result
 
 
+@app.patch(
+    "/api/update_assignments/{document_id}/reset", status_code=status.HTTP_200_OK
+)
+async def reset_map(document_id: str, session: Session = Depends(get_session)):
+    # drop all rows in assignments
+    stmt = delete(Assignments).where(Assignments.document_id == document_id)
```

Comment:
I: Return number of deleted records? `.returning(Assignments.geo_id)` then `n_deleted = len(stmt.fetchall())`. Not strictly necessary, but might be useful for the FE to be able to validate that the operation was successful.

Reply:
if we do the drop partition I don't think we get the number of records back.

Reply:
I believe that is correct